web/src/routes/__tests__/PoisPage.spec.tsx

```diff
@@ -72,11 +74,10 @@ describe('PoisPage', () => {
 
   it('should calculate correct language change paths', () => {
     mockUseLoadFromEndpointWithData(pois)
-    const { getAllByText, getByRole } = renderPois()
-    fireEvent.click(getByRole('link', { name: poi0.title }))
+    const { getAllByText, getByRole } = renderPois('/locations/test')
+    fireEvent.click(getByRole('button', { name: 'layout:changeLanguage' }))
+
     expect(getAllByText('English')[0]?.closest('a')).toHaveAttribute('href', poi0.availableLanguages.en)
     expect(getAllByText('Deutsch')[0]?.closest('a')).toHaveAttribute('href', poi0.availableLanguages.de)
-    // Pathname is not correctly updated, therefore the pathname does not include the slug
-    expect(getAllByText('اَللُّغَةُ اَلْعَرَبِيَّة')[1]?.closest('a')).toHaveAttribute('href', '/augsburg/ar/locations')
```

Comment:
Does anybody know why we're testing for the Arabic translation when there isn't one in the PoiModelBuilder?

Reply:
I would say that this is a poorly written test, that half-tests something that doesn't work (I also don't really understand the comment, how is the pathname not correctly updated? Is that because there is no Arabic translation in the poi?) If we want to test for Arabic, then I think we should add Arabic test data to the test